packages/costa/src/runnable.test.ts

```diff
@@ -119,4 +121,18 @@ describe('start runnable in normal way', () => {
     expect(cost < 5000);
     console.log(`child process exited after ${cost}ms`);
   });
+
+  it('should destroy the runnable when it blocks on loading plugin', async () => {
+    const logger = process;
+    const costa = new CostaRuntime(opts);
+    const r2 = new PluginRunnable(costa, logger);
+    await r2.bootstrap({ logger, pluginNotRespondingTimeout: 500 });
+    const simple = await costa.fetch(path.join(__dirname, '../tests/plugins/nodejs-not-responding'));
+    await costa.install(simple, process);
+    try {
+      await r2.start(simple, { foobar: true });
```

Comment:
We could use `expectAsync` instead to make sure the asynchronous error has been caught.